client/src/components/common/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,57 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";

```

Comment:
you can use standard material-ui way of styling

and generally remove library styled-components, it's used only in a few places in the project